WebContent/js/containers/ContentViewer.js

```diff
@@ -92,7 +100,22 @@ export class ContentViewer extends React.Component {
         dispatch(removeContent(removeIndex));
         // Do we need to change the selectedContent
         if (removeIndex <= selectedContent && selectedContent >= 1) {
-            dispatch(changeSelectedContent(selectedContent - 1));
+            const newTabIndex = selectedContent - 1;
+            dispatch(changeSelectedContent(newTabIndex));
+        }
+    }
+
+    handleKeyDown(e, index) {
```

Comment:
Should this method name be more specific, e.g "handleKeyDownOnContentTabLabel" ? In the future there could be more places within the contentViewer we have to handle keyDown events.

Reply:
renamed to suggested name